pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
```

Comment:
Thre are two schools of thought regarding the "Migrate" word:
Those implying copy in tech:
* https://discussions.apple.com/thread/4475182?tstart=0
* https://www.techopedia.com/definition/1180/data-migration.

But, still the inherent meaning of migrate is to transfer:
* http://www.thesaurus.com/browse/migrate

Hence, we should avoid the confusion. 
I also agree with the ambiguity of "one source into another".
Rephrase: 
'**Replicate textual information between different kinds of databases**'